workflows/toast_sim_ground.py

```diff
@@ -147,26 +167,33 @@ def load_instrument_and_schedule(args, comm):
 
     fname_pickle = (
         f"{os.path.basename(args.focalplane)}_"
-        f"thinfp={args.thinfp}_fsample={args.sample_rate}.pck"
+        f"thinfp={args.thinfp}_fsample={args.sample_rate}_fknee={args.fknee}.pck"
```

Comment:
Why not just pass a different file name as the `--focalplane` option?

Reply:
Agreed.